builtin/providers/google/resource_compute_instance.go

```diff
@@ -246,6 +246,20 @@ func resourceComputeInstance() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+
+			"automatic_restart": &schema.Schema{
```

Comment:
These should be in a nested structure called "scheduling" so we don't drift too far away from the official schema.